features-json/webgl2.json

```diff
@@ -260,7 +260,8 @@
       "57":"n"
     },
     "and_ff":{
-      "52":"n"
```

Comment:
@jdashg Sorry for the delay on this one! Can you replace the "and_ff" entry with just `"52":"y"` as it's not possible to add entries and only the latest version is tracked for this browser. Thanks!